new/bahn.py

```diff
@@ -1,24 +1,27 @@
 """
 Scraper for Deutsche Bahn parking lots
 
-This scraper is disabled unless you define your API token, either in
-- environment: e.g. `export BAHN_API_TOKEN=xxx` before running the scraper
-- or in a `.env` file in the root of the scraper package containing: `BAHN_API_TOKEN=xxx`
+This scraper is disabled unless you define your DB credentials, either in
+- environment: e.g. `export DB_CLIENT_ID=xxx && export DB_API_KEY=yyy` before running the scraper
+- or in a `.env` file in the root of the scraper package containing: `DB_CLIENT_ID=xxx DB_API_KEY=yyy`
 """
 
+import logging
 import warnings
-from typing import List
+from typing import List, Optional
 
 from decouple import config
 
 from util import *
 
-BAHN_API_TOKEN = config("BAHN_API_TOKEN", None)
+DB_CLIENT_ID = config("DB_CLIENT_ID", None)
+DB_API_KEY = config("DB_API_KEY", None)
 
-if not BAHN_API_TOKEN:
+
+if not DB_CLIENT_ID or not DB_API_KEY:
     warnings.warn(
         "Deutsche Bahn Parking API disabled! "
-        "You need to define BAHN_API_TOKEN in environment or in a .env file"
+        "You need to define DB_CLIENT_ID and DB_API_KEY in environment or in a .env file"
     )
 
 else:
```

Comment:
This script *cannot* function without `$DB_CLIENT_ID` & `$DB_API_KEY`, right?

If that is the case, I recommend aborting (exiting the process) with a non-0 status code, so that the calling script/user can react accordingly. Also, from the `warnings.warn()` docs (emphasis mine):

> Warning messages are typically issued in situations where it is useful to alert the user of some condition in a program, where **that condition (normally) doesn’t warrant raising an exception and terminating the program**.